st2api/tests/unit/controllers/v1/test_packs.py

```diff
@@ -169,6 +170,22 @@ def test_show(self):
         self.assertEqual(resp.status_int, 200)
         self.assertEqual(resp.json, PACK_INDEX['test2'])
 
+    def test_packs_register_endpoint_resource_register_order(self):
```

Comment:
This is not ideal, but I can't come up with a better test case which doesn't require bigger changes.